plugins/aggregators/histogram/histogram.go

```diff
@@ -119,6 +132,12 @@ func (h *HistogramAggregator) Description() string {
 
 // Add adds new hit to the buckets
 func (h *HistogramAggregator) Add(in telegraf.Metric) {
+	addTime := time.Time{}
+
+	if h.ExpirationInterval != 0 {
+		addTime = h.timeFun()
+	}
+
```

Comment:
Can we please do this in `Init()`?

Reply:
I saw this approach (`timeFunc` as a member of a struct and initialization in the constructor) inside [dropwizard parser implementation](https://github.com/influxdata/telegraf/blob/b2cfbb1acb6e3b191f3c6241e77457a52a645638/plugins/parsers/dropwizard/parser.go#L61) and use it here, to be more consistent over the whole codebase.
But it's not a big deal, I'll implement initialization of the `timefunc` in `init()`